src/arb/acb.jl

```diff
@@ -1281,6 +1282,29 @@ function moddelta(x::acb)
    return z
 end
 
+@doc Markdown.doc"""
+    modeisenstein(x::acb, k::Int)
+
+Return the non-normalized Eisenstein series $G_k(\tau)$ of
+$\mathrm{SL}_2(\mathbb{Z})$. Also defined for $\tau = i \infty$.
+"""
+function modeisenstein(x::acb, k::Int)
+  CC = parent(x)
+
+  k <= 2 && error("Eisenstein series are not absolute convergent for k = $k")
+  imag(x) < 0 && error("x is not in upper half plane.")
+  isodd(k) && return zero(CC)
+  imag(x) == Inf && return 2 * zeta(CC(k))
+
+  len = k ÷ 2 - 1
```

Comment:
Is this unicode? Is this ok? @thofma

Reply:
It's extended ascii, see 246 here: https://coding.tools/ascii-table.

Reply:
Can't enter it without using my mouse and copy & paste, so my answer would be no :(

Reply:
On my Linux system I only use AltGr+Shift+7 to enter it (like `/` but with AltGr), but this might be exclusive on Swedish keyboard?

Reply:
This gives me ½. It tried lots of combinations with `/` but could not make it work. Can you type it easily @tthsqe12?

Reply:
Hmm, lets just skip it then!

Reply:
> How do you type the division sign?
> Whilst holding down the [Alt] key, type [0247] on the numeric keypad (on the right). The alt code for the Divide symbol is 0247. Then release the [Alt] key. The Division (÷) symbol should appear in your document.

Utter rubbish. Does not work for me.